src/cljs/webchange/ui_framework/layout/navigation/svg.cljs

```diff
@@ -0,0 +1,10 @@
+(ns webchange.ui-framework.layout.navigation.svg)
```

Comment:
Let's keep icons in `webchange/ui_framework/components/icon` and use it with `icon` component.